Business Rules/RITM state change/Related_task_state_update.js

```diff
@@ -0,0 +1,16 @@
+//create business rule on sc_req_item table and add condtion 
+
+(function executeRule(current, previous /*null when async*/ ) {
+
+    var gr = new GlideRecord("sc_task");
+    gr.addQuery('request_item', current.sys_id);
+    gr.orderBy('order');
+    gr.setLimit(1);
+    gr.query();
+    if (gr.next() && gr.getValue('state') == 1) {
```

Comment:
If you added this condition to the query, you would not need to check the state for '1' (the magic number).